### ISSUE TITLE
Error: Protocol "https:" not supported. Expected "http:" when use `https` in ui5.url

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/karma-ui5#188]
When using the following ui5 config with karma-ui5,
```javascript
    ui5: {
      mode: 'script',
      url: 'https://sapui5.hana.ondemand.com/1.71.10',
      type: 'application',

      paths: {
        webapp: 'src/main/webapp/ui/nomination/js/talentpool/V2'
      },

      config: {
        bindingSyntax: 'complex',
        language: 'en_US',
        libs: 'sap.ui.core, sap.m, sap.ui.layout',
        theme: 'sap_belize',
        resourceroots: {
          'sap.sf.sm.test': '/base/src/test',
          'sap.sf.surj.shell': 'base/node_modules/@sf/sap.sf.surj.shell/src/sap/sf/surj/shell',
          'sap.sf.talentpool': 'base/node_modules/@sf/sap.sf.talentpool/src/sap/sf/talentpool'
        }
      },
      tests: ['sap/sf/sm/test/allTests']
    },
```
The console would print the following logs:
```
TypeError [ERR_INVALID_PROTOCOL]: Protocol "https:" not supported. Expected "http:"
    at new ClientRequest (_http_client.js:152:11)
    at Object.request (https.js:314:10)
    at Object.request (/Users/i307840/Workspace/bizx/au-sm/au-sm-web/node_modules/agent-base/patch-core.js:25:22)
    at Array.stream (/Users/i307840/Workspace/bizx/au-sm/au-sm-web/node_modules/http-proxy/lib/http-proxy/passes/web-incoming.js:126:74)
    at ProxyServer.<anonymous> (/Users/i307840/Workspace/bizx/au-sm/au-sm-web/node_modules/http-proxy/lib/http-proxy/index.js:81:21)
    at Framework.serveResources [as _serveResources] (/Users/i307840/Workspace/bizx/au-sm/au-sm-web/node_modules/karma-ui5/lib/framework.js:452:46)
    at /Users/i307840/Workspace/bizx/au-sm/au-sm-web/node_modules/karma-ui5/lib/framework.js:487:9
```

https://stackoverflow.com/questions/34147372/node-js-error-protocol-https-not-supported-expected-http

**Comment by matz3** [comment]
Thanks for reporting and sorry for the long delay. Will be fixed soon via #191

**Comment by matz3** [comment]
Should be fixed with v2.1.2
